doc/sphinx/source/howto/tune-for-deployment.rst

```diff
@@ -0,0 +1,192 @@
+.. tune-for-deployment.rst
+
+
+############################################
+Fine-tune code deployment to nGraph devices  
+############################################
+
+When deploying code to nGraph-enabled devices, there are some additional 
+configuration options that can be incorporated. Fine-tuning an nGraph-enabled
+device is as much of an art as it is a science; there are virtually limitless
+ways to command your training or inference computations to run the way you want.  
+
+In this section, which was written for framework architects or engineers who are 
+optimizing a new or less widely-supported framework, we provide some of our 
+learnings from the work we've done in developing our custom bridge code, such as
+that for our `ngraph tensorflow bridge`_ code. 
+
+Indeed, how this has worked :doc:`for many <../framework-integration-guides>` 
```

Comment:
The first couple times I read this sentence I couldn't figure out what it was saying. Now it's parsing for me, but it does have a complex structure. Maybe break it into two sentences?